server/proxy/server.js

```diff
@@ -58,7 +58,10 @@ app.get('/connect', (req, res) => {
             });
         }
     })
-    .catch((err) => res.sendStatus(400)); 
+    .catch((err) => {
+        console.error(err);
```

Comment:
Do we need this console?

Reply:
Yeah, that's how I managed to figure out what was wrong with the vote card test